lib/routes/dockerhub/repositories.ts

```diff
@@ -16,27 +15,28 @@ export const route: Route = {
 };
 
 async function handler(ctx: Context) {
-    const owner = ctx.req.param('owner').toLowerCase();
-    const limit = Number.parseInt(ctx.req.query('limit') || '10');
-    const link = `https://hub.docker.com/r/${owner}`;
-    const url = `https://hub.docker.com/v2/repositories/${owner}`;
-    const response = await got(url, {
-        searchParams: {
-            page_size: limit,
-        },
-    });
-    const item = response.data.results.map((repo) => ({
-        title: repo.name,
-        description: `${repo.description}<br>status: ${repo.status_description}<br>stars: ${repo.star_count}<br>pulls: ${repo.pull_count}`,
-        link: `https://hub.docker.com/r/${owner}/${repo.name}`,
-        author: owner,
-        pubDate: parseDate(repo.last_updated),
-        guid: `${owner}/${repo.name}`,
-    }));
+    const { owner, limit } = ctx.req.param();
```

Comment:
https://github.com/DIYgod/RSSHub/blob/be620d2348e10172a06eb0109f45845fdcb7149d/lib/routes/dockerhub/repositories.ts#L20
`limit` is a [query parameter](https://hono.dev/docs/api/request#query) not a path parameter.